src/index.js

```diff
@@ -149,6 +152,36 @@ module.exports = function(source, inputSourceMap) {
     options.sourceFileName = relative(options.sourceRoot, options.filename);
   }
 
+  if (this.version > 1 && options.presets) {
+    options.presets.forEach(preset => {
+      let name = preset;
+      let opts = {};
+
+      // if we have extra options put it in opts
+      if (Array.isArray(preset)) {
+        name = preset[0];
+        opts = preset[1];
+      }
+
+      if (
+        ["es2015", "env", "latest"].indexOf(name) > -1 &&
+        (!emitCache.has(options.presets) &&
+          (!opts ||
+            (opts && !opts.hasOwnProperty("modules")) ||
+            (opts && opts.modules)))
+      ) {
+        emitCache.set(options.presets, true);
+        this.emitWarning(
+          new Error(
+            `\n\n⚠️  Babel Loader\n
+It looks like your Babel configuration specifies a module transformer. Please disable it.
```

Comment:
Can we maybe give a little bit more information? Like "... . Please disable it, in order to allow webpack to create optimized bundles." Or "... . This disables tree shaking in webpack and will produce larger bundles. Please disable it."